.github/workflows/tests.yaml

```diff
@@ -10,6 +16,21 @@ on:
   workflow_dispatch: {}
 
 jobs:
+  installer-tests:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: checkout repository
+      uses: actions/checkout@v3
+
+    - name: run installer script
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
+      run: ./docs/install.sh
+
+    - name: run ktf
+      run: ~/.local/bin/ktf --help
```

Comment:
What is the purpose of `--help` here? 🤔

if the purpose is to validate that the ktf binary is present, please (at least) comment about the intent behind using `--help` in an automated workflow